controllers/namespace_controller_test.go

```diff
@@ -462,4 +476,60 @@ var _ = Describe("Namespace controller", func() {
 			HaveField("Annotations", HaveKeyWithValue("memo", "tama")),
 		))
 	})
+
+	Context("templated namespace", func() {
```

Comment:
What do you want this test to confirm?
I think confirming that ns is created from template and that labels and annotations are propagated has already been tested here.
https://github.com/cybozu-go/accurate/blob/cda331bc3d8386a655695191b419f22863d54db4/controllers/namespace_controller_test.go#L163

Reply:
This test is to ensure consistent reconciles. Reconcile loops is the one thing to fear when migrating to SSA.

Reply:
Thanks to you, I understood.